packages/user/FractalCore/ui/src/components/modals/set-min-scorers-modal.tsx

```diff
@@ -0,0 +1,76 @@
+import { useEffect } from "react";
+import { z } from "zod";
+import {
+    Dialog,
+    DialogContent,
+    DialogHeader,
+    DialogTitle,
+} from "@shared/shadcn/ui/dialog";
+import { useAppForm } from "@shared/components/form/app-form";
+import { useSetMinScorers } from "@/hooks/fractals/use-set-min-scorers";
+import { zU8 } from '@shared/lib/schemas/u8'
+import { MIN_MINIMUM_REQUIRED_SCORERS } from "@/lib/constants";
+import { useFractal } from "@/hooks/fractals/use-fractal";
+
+export const SetMinScorersModal = ({
+    show,
+    openChange,
+}: {
+    show: boolean;
+    openChange: (show: boolean) => void;
+}) => {
+    const { mutateAsync: setMinScorers } = useSetMinScorers();
+
+    const { data: fractal } = useFractal();
+    const form = useAppForm({
+        defaultValues: {
+            minScorers: fractal?.fractal?.minimumRequiredScorers || 4,
```

Comment:
I assume `minimumRequiredScorers` can never be `0`, but if that were a possible value, it would be overridden with `4` here. The `??` operator would be more precise here.